tests/unit/module_utils/test_ec2.py

```diff
@@ -394,6 +394,12 @@ def test_boto3_tag_list_to_ansible_dict(self):
         converted_dict = boto3_tag_list_to_ansible_dict(self.tag_example_boto3_list)
         self.assertEqual(converted_dict, self.tag_example_dict)
 
+    def test_boto3_tag_list_to_ansible_dict_empty(self):
+        # AWS returns [] when there are no tags
+        self.assertEqual(boto3_tag_list_to_ansible_dict([], {}))
+        # Minio returns [{}] when there are no tags
+        self.assertEqual(boto3_tag_list_to_ansible_dict([{}], {}))
```

Comment:
```suggestion
        self.assertEqual(boto3_tag_list_to_ansible_dict([{}]), {})
```

Reply:
Sorry, i must have been blind.

Reply:
easy mistake to make, nothing to worry about.